WowPacketParserModule.VV2_4_3_8606/WowPacketParserModule.VV2_4_3_8606.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
look at filename here, pretty sure this isnt right

Reply:
Yup.... This is from where I made a wrong project. Removing it.